### PR TITLE
#4: Not that much, more might come, who knows...

### PR DESCRIPTION
**Pull request by tangalbert919**
I'm afraid you'll have to decide whether or not the new stuff in the weeb cog will be for upvoters or patrons or not. :frowning:

### CODE REVIEW

File: cogs/weeb.py [hunk 1]
```diff
@@ -70,6 +70,26 @@ def __init__(self, bot):
         embed.set_image(url=url)
         embed.set_footer(text="Powered by weeb.sh")
         await ctx.send(embed=embed)
+    
+    @commands.command()
+    async def lewd(self, ctx):
+        """What a lewd!"""
+        if ctx.message.channel.is_nsfw():
+            async with self.bot.session.get('https://api-v2.weeb.sh/images/random?type=lewd&nsfw=true', headers={'Authorization': self.bot.settings.weebtoken, 'User-Angent': 'Monika/1.0.0'}) as url:
```
Comment (zspri) [inline cogs/weeb.py]: 'User-Agent' is spelled wrong.

File: cogs/weeb.py [hunk 2]
```diff
@@ -70,6 +70,26 @@ def __init__(self, bot):
...
+                url = await url.json()
+                url = url.get("url")
+        else:
+            async with self.bot.session.get('https://api-v2.weeb.sh/images/random?type=lewd', headers={'Authorization': self.bot.settings.weebtoken, 'User-Angent': 'Monika/1.0.0'}) as url:
```
Comment (zspri) [inline cogs/weeb.py]: 'User-Agent' is spelled wrong.

File: cogs/fun.py [hunk 3]
```diff
@@ -69,21 +69,23 @@ def __init__(self, bot):
...
-            return
-        elif username.id == 201745963394531328 or username.id == 206197394667208704:
-            await ctx.send("That's a funny joke, {}.".format(ctx.message.author.name))
-            return
-        elif username.id == 399315651338043392:
-            await ctx.send("You're so funny, {}.".format(ctx.message.author.name))
-            return
         try:
-            await ctx.send("``characters/{}.chr`` deleted successfully.".format(username.name.lower()))
+            username: discord.Member
```
Comment (zspri) [inline cogs/fun.py]: Is this correct Python syntax?

**Review by zspri — CHANGES_REQUESTED** [review]
Mostly looks good, except for a few misspellings and such. Not entirely sure if we want to add this yet(?)

**Comment by tangalbert919** [comment]
First 2 comments: I'll fix that.
Last one: Yes. I tested that.

**Review by zspri — APPROVED** [review]

File: cogs/fun.py (hunk 3 above)
Comment (kanielyaniel) [reply cogs/fun.py]: no

Comment (tangalbert919) [reply cogs/fun.py]: I just tested that one line. It works fine.

EDIT: I did set up this bot and get it working before I did changes. I'm not going to blunder in to a Discord bot and edit it without testing it.

**Comment by kanielyaniel** [comment]
I don't believe the lewd type should be its own command as it's just really some meta stuff and not exactly up to the weeb.sh command expectations for me.

**Review by kanielyaniel — APPROVED** [review]
gud

**Pull request merged**